app/client/src/features/form/components/fragments/Award.js

```diff
@@ -3,7 +3,8 @@
  */
 
 import React from 'react'
-import { Divider, LabeledInput } from '../../../../common/components'
+import { Divider } from '../../../../common/components'
+import LabeledInput from '../fragments/LabeledInput'
```

Comment:
Instead of going back one folder and reentering the folder, you should just be able to do

```js
import LabeledInput from './LabeledInput'
```

The same goes for all the other files in `fragments` where `LabeledInput` is imported.